client/plots/boxplot/viewModel/LegendDataMapper.ts

```diff
@@ -23,21 +27,22 @@ export class LegendDataMapper {
 		if (config.term.q?.descrStats) {
 			this.legendData.push({
 				label: `Descriptive Statistics${isTerm2 ? `: ${config.term.term.name}` : ''}`,
-				items: config.term.q.descrStats
+				items: this.setDescrStatArr(config.term.q.descrStats)
 			})
 		}
 		if (isTerm2 && isTerm2.q?.descrStats) {
```

Comment:
why is `isTerm2` variable required, and not simply use optional chaining `if (config?.term2?.q?.descrStats)`?